### PR TITLE
Add '/v2/transactions/{txid}' endpoint.

### PR DESCRIPTION
**Pull request by winder**

### CODE REVIEW

**Comment by brianolson** [comment]
This feature should not be done. txid must not be encouraged as a lookup for transactions. #164 should be closed.

File: api/indexer.oas2.json [hunk 1]
```diff
@@ -470,6 +475,37 @@
         }
       }
     },
+    "/v2/transactions/{txid}": {
+      "get": {
+        "description": "Lookup transactions.",
```
Comment (rotemh) [inline api/indexer.oas2.json:480]: Consider changing to "lookup a single transaction"

File: api/indexer.oas2.json [hunk 2]
```diff
@@ -470,6 +475,37 @@
...
+        "operationId": "lookupTransactions",
+        "parameters": [
+          {
+            "type": "string",
+            "name": "txid",
+            "in": "path",
+            "required": true
+          }
+        ],
+        "responses": {
```
Comment (rotemh) [inline api/indexer.oas2.json:499]: Seems like it's missing the 404

**Review by rotemh — APPROVED** [review]
Two small comments. Otherwise looks great!

**Review by algorotem — APPROVED** [review]

**Pull request merged**